components/FormField/src/index.tsx

```diff
@@ -1,5 +1,22 @@
-import { FormField, FormFieldProps } from '@utrecht/component-library-react';
+import React, { ForwardedRef, forwardRef, PropsWithChildren } from 'react';
+import {
+  FormField as UtrechtFormField,
+  FormFieldProps as UtrechtFormFieldProps,
+} from '@utrecht/component-library-react';
 import './index.scss';
+import clsx from 'clsx';
+
+interface FormFieldProps extends PropsWithChildren<UtrechtFormFieldProps> {
+  flex?: boolean;
```

Comment:
Weet niet of ik css styling als prop zou meegeven. Ik zou de props vooral gebruiken voor functionaliteit en classes voor styling. Wat vind jij?

Reply:
We kunnen het beter als classname meegeven inderdaad. Al zit dit een beetje tussen styling en functionaliteit in misschien? Want het is naast elkaar tonen vs onder elkaar.

Maar ik zal de classname anders even in de documentatie zetten in storybook.

Reply:
Done!